lib/common/monitoring/alarms/LatencyAlarmFactory.ts

```diff
@@ -136,6 +136,11 @@ export interface LatencyThreshold extends CustomAlarmThreshold {
   readonly maxLatency: Duration;
 }
 
+export interface LatencyTimeoutPercentageThreshold
+  extends CustomAlarmThreshold {
+  readonly maxLatencyPercentage: number;
```

Comment:
By itself without seeing the interface name, it's unclear what this actually means.

Maybe `maxLatencyPercentageOfTimeout`?